generator/gm/gm_global_a_powertrain.dbc

```diff
@@ -210,13 +210,17 @@ BO_ 810 TCICOnStarGPSPosition: 8 K73_TCIC
  SG_ GPSLongitude : 39|32@0+ (1,-2147483648) [0|0] "milliarcsecond"  NEO
  SG_ GPSLatitude : 7|32@0+ (1,0) [0|0] "milliarcsecond"  NEO
 
-BO_ 840 EBCMWheelSpdFront: 4 K17_EBCM
+BO_ 840 EBCMWheelSpdFront: 5 K17_EBCM
  SG_ FLWheelSpd : 7|16@0+ (0.0311,0) [0|255] "km/h"  NEO
  SG_ FRWheelSpd : 23|16@0+ (0.0311,0) [0|255] "km/h"  NEO
 
 BO_ 842 EBCMWheelSpdRear: 5 K17_EBCM
  SG_ RLWheelSpd : 7|16@0+ (0.0311,0) [0|255] "km/h"  NEO
  SG_ RRWheelSpd : 23|16@0+ (0.0311,0) [0|255] "km/h"  NEO
+ SG_ MovingForward : 32|1@0+ (1,0) [0|1] "" XXX
+ SG_ MovingBackward : 33|1@0+ (1,0) [0|1] "" XXX
+ SG_ MovingForward2 : 35|1@1+ (1,0) [0|1] "" XXX
+ SG_ MovingBackward2 : 36|1@0+ (1,0) [0|1] "" XXX
```

Comment:
Not sure what the difference is between the alt signals, they're mostly identical except for falling edge timings. Could be related to left/right or forward/rear wheels but I couldn't find a correlation